### ISSUE TITLE
How to use fairseq for tagging?

### ISSUE THREAD

**Issue by gagb** [pytorch/fairseq#344]
Is it possible to use this library to trained a tagger, for example, a POS tagger?

**Comment by gagb** [comment]
Yes, you are right. But tagging requires the output sequence to be of the same length as the input.
And that is what I am confused about.

Sorry, I should have been more specific.

**Comment by mohammadKhalifa** [comment]
@gagb 
you can use this [fork](https://github.com/mohammadKhalifa/fairseq-tagging) adapted for sequence tagging
